wrappers/python/c_files.cpp

```diff
@@ -139,6 +139,73 @@ void init_c_files(py::module &m) {
         .def_readwrite("angular_acceleration", &rs2_pose::angular_acceleration, "X, Y, Z values of angular acceleration, in radians/sec^2")
         .def_readwrite("tracker_confidence", &rs2_pose::tracker_confidence, "Pose confidence 0x0 - Failed, 0x1 - Low, 0x2 - Medium, 0x3 - High")
         .def_readwrite("mapper_confidence", &rs2_pose::mapper_confidence, "Pose map confidence 0x0 - Failed, 0x1 - Low, 0x2 - Medium, 0x3 - High");
+
+    py::class_<sc_float2> float2(m, "float2"); // No docstring in C++
```

Comment:
Let's try to add tests for it.
We feel the preset fields with real values
write it
get it
and compare to the origin

Reply:
working on this